gate-web/src/main/groovy/com/netflix/spinnaker/gate/config/GateConfig.groovy

```diff
@@ -387,41 +370,27 @@ class GateConfig extends RedisHttpSessionConfiguration {
    * Additionally forwards request origin metadata (deck vs api).
    */
   @Bean
-  FilterRegistrationBean authenticatedRequestFilter() {
+  FilterRegistrationBean<AuthenticatedRequestFilter> authenticatedRequestFilter() {
     // no need to force the `AuthenticatedRequestFilter` to create a request id as that is
     // handled by the `RequestTimingFilter`.
-    def frb = new FilterRegistrationBean(new AuthenticatedRequestFilter(true, true, false, false))
+    def frb = new FilterRegistrationBean<>(new AuthenticatedRequestFilter(true, true, false, false))
     frb.order = Ordered.LOWEST_PRECEDENCE - 1
     return frb
   }
 
-  /**
-   * This pulls the `springSecurityFilterChain` in front of the {@link AuthenticatedRequestFilter},
-   * because the user must be authenticated through the security filter chain before their username/credentials
-   * can be pulled and forwarded in the AuthenticatedRequestFilter.
-   */
-  @Bean
-  FilterRegistrationBean securityFilterChain(
```

Comment:
This is now handled in `application.properties` since all it was doing was reordering the spring security filter. That's a supported config property in Spring Boot, so no need to expose internal details of Spring Security!